provider/resource_keycloak_openid_client_authorization_client_policy.go

```diff
@@ -0,0 +1,200 @@
+package provider
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakOpenidClientAuthorizationClientPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientAuthorizationClientPolicyCreate,
+		Read:   resourceKeycloakOpenidClientAuthorizationClientPolicyRead,
+		Delete: resourceKeycloakOpenidClientAuthorizationClientPolicyDelete,
+		Update: resourceKeycloakOpenidClientAuthorizationClientPolicyUpdate,
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakOpenidClientAuthorizationClientPolicyImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"resource_server_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"decision_strategy": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"owner": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"logic": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"policies": {
```

Comment:
Is the `policies` attribute needed for this policy?  I thought it was only needed for the aggregate policy resource.